rancher/rancher-compose/rancher-services/index.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Custom Rancher Services
+layout: rancher-default
+
+---
+
+## Custom Rancher Services
+
+Custom Rancher services are configured by using a special image name in the compose template.  The image name is how `rancher-compose` knows to set up a Rancher service versus a normal service.
+
+Service | Image Name
+--------|-----------
+Load Balancer | rancher/load-balancer-service
+External Service | rancher/external-service
+Alias/DNS Service | rancher/dns-service
+
+### Load Balancer
+
+A load balancer can be scheduled like any other service. Read more about [scheduling]({{site.baseurl}}/rancher/rancher-compose/scheduling/) services and load balancers using `rancher-compose`.
+
+Load balancing using the basic use case of load balancers is simply adding ports and linking services. Any traffic directed to a source port will be sent to the private port of the linked services.
+
+We go into more detail on [advanced routing options]({{site.baseurl}}/rancher/rancher-ui/applications/stacks/adding-balancers/#advanced-routing-optons) for load balancers in our UI section, but anything that we can create in Rancher can be created using `rancher-compose`.
+
+Advanced routing options use `labels` in the `docker-compose.yml` file. Here's the basic syntax for applying the label. You would only use this label if you were going to do some advanced routing rules.
+
+Label Key | Label Value
+---| ---
+`io.rancher.loadbalancer.target.SERVICE_NAME` | `REQUEST_HOST:SOURCE_PORT/REQUEST_PATH=TARGET_PORT`
```

Comment:
We perhaps should mention that all of these parameters optional first, and then list all the possible combinations